web/src/games/cardtable/game.test.ts

```diff
@@ -0,0 +1,231 @@
+import { Client } from 'boardgame.io/client';
+import { CardTableGame } from './game';
+
+describe('deck moves', () => {
+  test('cutDeck(1) should rotate deck by one card', () => {
+    let client = Client({ game: CardTableGame });
+    client.moves.cutDeck(1);
+    const { G } = client.store.getState();
+    expect(G.deck[0]).toEqual({ id: 'AC', rank: 26, faced: false, img: './media/png/AC.png' });
```

Comment:
The game rules (game.ts) should be independent of implementation details. The filepath is a implementation detail, so please remove the field "img" from this type and derive it when needed from the id.

Reply:
This is a test scenario, the file is a utility for loading game state in the game.test.  

The data model for the game is still in flux as many parts will be pared away -- would like to defer the decision on the card.ID property as the name is coincidental and not purposely matching the glyphs for the deck (which is a free distribution from American Contract Bridge League (ACBL) don't know if they're appropriate for copyleft distribution.

Reply:
but in general you dont want a file path as part of your game state. The game state is stored on databases after a match is played. What if you need to change the file path later, or change the image? You would have the stale file path stored on the database for all past games, and it would break the game for users coming back to them.

If ID is not enough, create another enum or something that would be enough. But **do not** store filepaths here.